scripts/install

```diff
@@ -216,6 +216,35 @@ newuser.confirmation_token = nil
 newuser.save
 ApplicationSetting.last.update_attributes(password_authentication_enabled_for_web: $use_web_account, signup_enabled: $use_web_account)" | gitlab-rails console
 
+
+#=================================================
+# DEFINE THE BACKUP STRATEGY: https://docs.gitlab.com/ce/raketasks/backup_restore.html#excluding-specific-directories-from-the-backup
+#=================================================
+
+backup_db=1
```

Comment:
Here and as well into the upgrade script, you don't have to set the variable if it's not used later in the script.
That's not an issue, but only that you can spare those lines.

Reply:
Done